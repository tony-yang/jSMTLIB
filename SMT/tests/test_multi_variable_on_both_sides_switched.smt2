(set-logic QF_LRA)
(set-info :source | SMT-COMP'06 organizers |)
(set-info :smt-lib-version 2.0)
(set-info :category "check")
(set-info :status sat)
(set-info :notes |This benchmark is designed to check if the DP supports bignumbers.|)
(declare-fun x1 () Real)
(declare-fun x2 () Real)
(declare-fun x3 () Real)
(assert (< (+ 6 (* 5 x3)) (+ (- x1) (* 3 x2))))
(check-sat)
(exit)
