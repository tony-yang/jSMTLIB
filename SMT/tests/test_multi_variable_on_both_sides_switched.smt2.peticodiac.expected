# set_logic: QF_LRA
# :smt-lib-version: 2.0
# :category: "check"
# :status: sat
# :notes: |This benchmark is designed to check if the DP supports bignumbers.|
p cnf 3 1
c -1.0 3.0 -5.0
b 3 >:6.0 NO_BOUND
